src/org/traccar/database/ConnectionManager.java

```diff
@@ -41,6 +41,7 @@
 
     private final long deviceTimeout;
     private final boolean enableStatusEvents;
+    private final boolean correctMotionState;
```

Comment:
What does this parameter do? I think we need a better name because it's not very clear.